src/file_util.py

```diff
@@ -59,7 +59,7 @@ def Remove(path):
   if sys.platform == 'win32':
     # shutil.rmtree() may not work in Windows if a directory contains read-only
     # files.
-    proc.check_call('rmdir /S /Q "%s"' % path, shell=True)
+    proc.check_call(['rmdir', '/S', '/Q', path], shell=True)
```

Comment:
I think `shell=True` doesn't really work with the list form for check_call. I think we just want the `SpecialCases` part.

Reply:
Done